checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitAnnotatedTypeFormatter.java

```diff
@@ -51,8 +51,7 @@ public String visitNull(AnnotatedNullType type, Set<AnnotatedTypeMirror> visitin
             if (type.getAnnotation(Nullable.class) != null) {
                 // The null type will be understood as nullable by readers (I hope), therefore omit
                 // the annotations if they are @Nullable.
-                // Note: The visitTypeVariable will still print lower bounds with Null kind as
-                // "Void"
+                // Note: The visitTypeVariable will print lower bounds with Null kind as "NullType"
                 if (!currentPrintInvisibleSetting) {
                     return "null (NullType)";
```

Comment:
```suggestion
                    return "null (@Nullable NullType)";
```

If we want to keep the parenthetical, I think this would be more explicit.